components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -16428,6 +16428,8 @@ public void addAPIRevision(APIRevision apiRevision) throws APIManagementExceptio
                 String driverName = connection.getMetaData().getDriverName();
                 if (driverName.contains("Oracle")) {
                     getClientCertificatesQuery = SQLConstants.APIRevisionSqlConstants.GET_CLIENT_CERTIFICATES_ORACLE_SQL;
+                } else if (driverName.contains("Microsoft JDBC")) {
+                    getClientCertificatesQuery = SQLConstants.APIRevisionSqlConstants.GET_CLIENT_CERTIFICATES_MSSQL;
```

Comment:
Don't we need to mention as below: ?

https://github.com/wso2/carbon-apimgt/blob/dc0af27d72fcc1c9c63b056266ff06e221ee2dca/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java#L16236-L16237